app/javascript/app/components/button-group/button-group-component.jsx

```diff
@@ -11,8 +11,14 @@ import PropTypes from 'prop-types';
 
 import styles from './button-group-styles.scss';
 
-const ButtonGroup = ({ className, onInfoClick }) => (
-  <div className={cx(styles.buttonGroup, className)}>
+const ButtonGroup = ({ className, onInfoClick, disabled }) => (
+  <div
+    className={cx(
+      styles.buttonGroup,
+      disabled ? styles.disabled : '',
+      className
+    )}
+  >
     <Button className={cx(styles.button, styles.first)} onClick={onInfoClick}>
```

Comment:
We may use a  disabled={!onInfoClick} here to show the not allowed hover also for if the info button is not enabled

Reply:
I am not seeing these at all on my branch, and I remember fixing them in a previous PR. Could you delete the branch and re pull (using --prune, incase of duplicate name?